### PR TITLE
More `DataFrame` translations

### PR DESCRIPTION
**Pull request by vincentarelbundock**
Polars methods

- [x] `unique`
- [x] `fill_null`
- [x] `drop`
- [x] `drop_in_place`
- [x] `drop_nulls`
- [x] `frame_equal`
- [x] `shift`
- [x] `shift_and_fill`
- [x] `quantile`
- [x] `fill_nan`

S3 methods

- [x] `unique`
- [x] `na.omit`

Will not do this in the current PR after all:

- [x] `sort`
- [ ] `is_duplicated`
- [ ] `is_unique`

### CODE REVIEW

**Comment by sorhawell** [comment]
this is amazing :) I didn't get to complete my review but it looks very good. I will continue tomorrow night.

**Comment by sorhawell** [comment]
almost there still missing some tests and more examples

**Comment by vincentarelbundock** [comment]
Do you plan to work on those or would you like me
to take a look.

No rush or pressure either way.

**Comment by sorhawell** [comment]
> Do you plan to work on those or would you like me to take a look?

I would be happy for that.

By the way. What do you think of the wording of docs @param `by` and `...`? I think it is strange to an average R user, but fairly common for a python or rust user. I'm not sure how to state it intuitively.
Iterable Into<Expr> in practice means some R object that `lapply()` can iterate over and which is either already an Expr or can be converted with `pl$lit()` and/or `pl$col()`without errors.

**Comment by vincentarelbundock** [comment]
I added some tests and lots of examples. I also tried to simplify the `by` argument documentation, but I don't think my version is great either. But I think the numerous examples should help clarify things for most users.

**Comment by sorhawell** [comment]
@vincentarelbundock  great this is a huge contribution :) I will merge in after checks

**Review by sorhawell — APPROVED** [review]

**Comment by sorhawell** [comment]
@vincentarelbundock likely after merging #140  and #146 into this PR failed. I tried to resolve it but think the solution drifted away from you vision of the s3 methods.

Can you solve the errors?

It might also be better for LazyFrame to not implement the "forbidden" collect methods nrow, dim, ... than throwing an R error. Some auto completion method in an IDE might call e.g. dim on any method.

**Comment by vincentarelbundock** [comment]
Should be good to go now.

File: R/expr__expr.R
Comment (eitsupi) [inline R/expr__expr.R:63]: I do not understand the need for this.

**Pull request merged**
